### PR TITLE
Feature/devops 483

### PR DESCRIPTION
**Pull request by bmcbm**
Use ubuntu:xenial 

Remove runit.sh 

Add some logging (Work in progress) 

Use CMD to start nginx

### CODE REVIEW

**Comment by bmxlugo** [comment]
:+1:

File: runit.sh [hunk 1]
```diff
@@ -1,12 +0,0 @@
-#!/bin/bash
```
Comment (bmxdasz) [inline runit.sh]: What user will start nginx? Now we have configured this for example on Jenkins.

Comment (bmxlugo) [inline runit.sh]: @bmcbm i forgot about that, Dawid has right we need to have www-data access

Comment (bmcbm) [inline runit.sh]: @bmxdasz : Will NginX **write** data? 

Can you point me to a jenkins job that I can take a lok at?

Comment (bmxdasz) [inline runit.sh]: Nginx write data into cache and logs (app/cache, app/logs).

Example job: https://jenkins.bemit.dk/job/bm.app.bdk/configure

Comment (bmxlugo) [inline runit.sh]: @bmcbm what about adding that changes to PR that Dawid wrote. So we can merge PR.

**Pull request merged**